reloj-checador/app/views/reports/absence_by_month.html.erb

```diff
@@ -1,56 +1,56 @@
-            <table class="table table-striped table-bordered">
-                <thead>
-                    <tr>
-                        <th class="text-center">Fecha</th>
-                        <th class="text-center">Compañía</th>
-                        <th class="text-center">Entradas Registradas</th>
-                        <th class="text-center">Ausencias</th>
-                    </tr>
-                </thead>
-                <tbody>
-                    <% @abscenes.each do |abscene| %>
+        <form class="col-12 col-sm-12 col-md-3" action="<%= reports_absence_by_month_path %>" method="get">
+            <div class="row align-content-end ml-0 ml-sm-4">
+                <div class="form-group">
+                    <input type="date" class="form-control" name="date" value="<%= params[:date] %>">
+                </div>
+                <div class="form-group">
+                    <button class="btn btn-secondary btn-sm ml-0 ml-3">BUSCAR</button>
+                </div>
+            </div>
+        </form>
+        <div class="col-12 col-sm-12 col-md-10">
+            <br>
+            <div class="table-responsive">
+                <table class="table table-striped table-bordered">
+                    <thead>
                         <tr>
-                            <td class="text-center"><%=  abscene.date %></td>
-                            <td class="text-center"><%=  abscene.count %></td>
-                            <td class="text-center"><%=  0 %></td>
-                            <% emp_count = 0 %>
-                            <td class="text-center"><%=  (emp_count > 0) ? emp_count : "0" %></td>
+                            <th style="text-align: center;">Fecha</th>
+                            <th style="text-align: center;">Compañía</th>
+                            <th style="text-align: center;">Entradas Registradas</th>
+                            <th style="text-align: center;">Ausencias</th>
                         </tr>
-                    <% end %>
-                </tbody>
-            </table>
-            <div class="media mb-4">
-                <div class="media-body">
-                    <%= will_paginate @abscenes %>
+                    </thead>
+                    <tbody>
+                        <% @abscenes.each do |abscene| %>
+                            <% company = @companies.select{|company| company if company.company_id == abscene.company_id}.first %>
+                            <% abscene_count = company.emp_count - abscene.emp_count %>
+                            <tr>
+                                <td style="text-align: center;"><%=  abscene.date %></td>
+                                <td style="text-align: center;"><%=  company.name %></td>
+                                <td style="text-align: center;"><%=  abscene.emp_count %></td>
+                                <td style="text-align: center;"><%=  (abscene_count > 0) ? abscene_count : "0" %></td>
```

Comment:
Maybe is better make an sass archive with only an varible like $text-center and use it in all the elements that u need